### DISCUSSION TITLE
aiocqhttp 或者 nonebot 为什么不用 aiohttp来做呢？

### DISCUSSION THREAD

**Discussion by ilihu** [nonebot/discussions#356] [想法]
aiocqhttp 或者 nonebot 为什么不用 aiohttp来做呢？ aiohttp也有 提供http server的功能啊，为什么要用一个重复的轮子 Quart 呢？

**Comment by yanyongyu** [comment]
aiohttp既不是wsgi也不是asgi，独树一帜，aiohttp才是最怪的那个

**Reply by ilihu** [reply]
aiohttp 不是 asgi 么，为什么说不是 asgi 呢？

**Reply by yanyongyu** [reply]
https://asgi.readthedocs.io/en/latest/
